include/classes/Utility.class.php

```diff
@@ -318,9 +318,9 @@ public static function sanitizeIntArray($int_array) {
 		
 		$sanitized_array = array();
 		foreach ($int_array as $i => $value) {
-			$sanitized_array[$i] = intval($value);
+			$sanitized_array[$i] = array('filter' => FILTER_VALIDATE_INT, 'flags'  => FILTER_REQUIRE_SCALAR,);
 		}
-		return $sanitized_array;
+		return filter_var_array($int_array, $sanitized_array);

```

Comment:
It seems this whole function can be replaced by `array_filter($int_array, 'is_int')`. Can you please double check? Thanks.

Reply:
I just checked I think it is more preferable. will implement it right away

Reply:
I do the same particular error, whenever I am making use of ```array_filter``` and ```array('filter...``` , therefore I think the use of ```intval``` should be continue